### PR TITLE
LocalDataBase

### PR DESCRIPTION
**Pull request by TanyaStk**

### CODE REVIEW

File: Netflix/UseCases/LocalDataSourceUseCase.swift [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+//  Created by Tanya Samastroyenka on 17.08.2022.
+//
+
+import Foundation
+import UIKit
+import CoreData
+import RxSwift
+
+protocol LocalDataSourceProtocol {
+    func saveMoviesFor(categoryName: String, models: [Movie]) throws
```
Comment (dnv190) [inline Netflix/UseCases/LocalDataSourceUseCase.swift]: Enum?

File: Netflix/CoreData/Entities/MovieEntity+CoreDataProperties.swift [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+    @NSManaged public var id: Int64
+    @NSManaged public var isFavorite: Bool
+    @NSManaged public var overview: String?
+    @NSManaged public var posterPath: String?
+    @NSManaged public var releaseData: String?
+    @NSManaged public var runtime: Int64
+    @NSManaged public var title: String?
+    @NSManaged public var voteAverage: String?
+    @NSManaged public var category: CategoryEntity?
```
Comment (dnv190) [inline Netflix/CoreData/Entities/MovieEntity+CoreDataProperties.swift]: [Array] 🔴 
[-----]

File: Netflix/CoreData/Entities/LatestMovieEntity+CoreDataProperties.swift [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+//  Netflix
+//
+//  Created by Tanya Samastroyenka on 18.08.2022.
+//
+//
+
+import Foundation
+import CoreData
+
+extension LatestMovieEntity {
```
Comment (dnv190) [inline Netflix/CoreData/Entities/LatestMovieEntity+CoreDataProperties.swift]: [Optional]
♻️

**Comment by TanyaStk** [comment]
DataBaseScheme

**Review by dnv190 — APPROVED** [review]

**Pull request merged**